admin/app/dfp/DfpTemplateCreativeCacheJob.scala

```diff
@@ -15,6 +16,11 @@ class DfpTemplateCreativeCacheJob(dfpApi: DfpApi) {
       val threshold = GuCreative.lastModified(cached) getOrElse now.minusMonths(1)
       val recentlyModified = dfpApi.readTemplateCreativesModifiedSince(threshold)
       val merged = GuCreative.merge(cached, recentlyModified)
-      Store.putDfpTemplateCreatives(Json.stringify(Json.toJson(merged)))
+      val wrapper = GuCreativeWrapper(
+        updatedTimeStamp = DateTime.now().toString,
+        creatives = merged,
+      )
+      Store.putDfpTemplateCreatives(Json.stringify(Json.toJson(wrapper)))
```

Comment:
I thought the issue with changing the schema in the typescript jobs was that Frontend has started using it already? But it's putting the file here so it can't be?

So there's no need to update this...?